pkg/common-controller/groupsnapshot_controller_helper.go

```diff
@@ -567,13 +567,36 @@ func (ctrl *csiSnapshotCommonController) updateGroupSnapshotStatus(groupSnapshot
 	var pvcVolumeSnapshotRefList []crdv1alpha1.PVCVolumeSnapshotPair
 	if groupSnapshotContent.Status != nil && len(groupSnapshotContent.Status.PVVolumeSnapshotContentList) != 0 {
 		for _, contentRef := range groupSnapshotContent.Status.PVVolumeSnapshotContentList {
-			groupSnapshotContent, err := ctrl.contentLister.Get(contentRef.VolumeSnapshotContentRef.Name)
+			pv, err := ctrl.client.CoreV1().PersistentVolumes().Get(context.TODO(), contentRef.PersistentVolumeRef.Name, metav1.GetOptions{})
+			if err != nil {
+				klog.V(5).Infof(
+					"updateGroupSnapshotStatus[%s]: unable to get PV [%s] from the API server",
+					utils.GroupSnapshotKey(groupSnapshot),
+					contentRef.PersistentVolumeRef.Name)
+			}
+
+			pvcName := ""
+			if pv != nil {
+				if pv.Spec.ClaimRef != nil {
+					pvcName = pv.Spec.ClaimRef.Name
+				} else {
+					klog.V(5).Infof(
+						"updateGroupSnapshotStatus[%s]: PV [%s] is not bound",
+						utils.GroupSnapshotKey(groupSnapshot),
+						contentRef.PersistentVolumeRef.Name)
+				}
+			}
+
+			volumeSnapshotContent, err := ctrl.contentLister.Get(contentRef.VolumeSnapshotContentRef.Name)
 			if err != nil {
 				return nil, fmt.Errorf("failed to get group snapshot content %s from group snapshot content store: %v", contentRef.VolumeSnapshotContentRef.Name, err)
 			}
 			pvcVolumeSnapshotRefList = append(pvcVolumeSnapshotRefList, crdv1alpha1.PVCVolumeSnapshotPair{
 				VolumeSnapshotRef: v1.LocalObjectReference{
-					Name: groupSnapshotContent.Spec.VolumeSnapshotRef.Name,
+					Name: volumeSnapshotContent.Spec.VolumeSnapshotRef.Name,
+				},
+				PersistentVolumeClaimRef: v1.LocalObjectReference{
+					Name: pvcName,
```

Comment:
Yes, it will. And this is equivalent to an empty `LocalObjectReference`.
And since the `VolumeSnapshotRef` itself is marked as `omitempty`, it will be omitted.

But I realized now this is not evident from reading the code. Let me express it more clearly.